pep/app/styles/_utilities.scss

```diff
@@ -73,3 +73,14 @@
 .highlight-odd li:nth-child(odd) {
     background-color: $color-backdrop;
 }
+
+// ---------------------------
+// Text / Font
+// ---------------------------
+.text-justify {
+    text-align: justify !important;
+}
+
+.text-indent-none {
+    text-indent: 0 !important;
+}
```

Comment:
I think we'd want to keep the existing text indenting in place, as justified text can still have paragraph indenting. Maybe confirm w/Neil, but his bug description also did show the current indenting preserved in his example screenshot and styles.

Reply:
@nrshapiro Would you like the `Justified` text alignment to be like your first picture or second picture in #228 ?